### DISCUSSION TITLE
Can't SSH into docker LCX host

### DISCUSSION THREAD

**Discussion by rossdargan** [tteck/Proxmox#117] [General]
I used the script to spin up the docker host and that worked really well (thanks!). From the website termnial I can log in, but remotely I just get permission denied. I suspect this is a general linux thing and I'm just being an idiot but if you could help me that would be super helpful!

Thanks

**Comment by tteck** [comment]
You are trying to SSH in?

**Comment by rossdargan** [comment]
yes

**Reply by tteck** [reply]
I renamed the issue. I'm sure someone that knows SSH will chime in and help ya. 🤞🏻

**Reply by FredericMa** [reply]
I had the same issue when trying to transfer files and enabling root login for sshd fixed it:
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/v2v_guide/preparation_before_the_p2v_migration-enable_root_login_over_ssh
If you created the container with auto-login you might need to set the root password first via `passwd`.

I guess created users, other then root, should be able to remote login without any issues.

**Reply by rossdargan** [reply]
This absolutely was the issue. Thankyou.

**Comment by rossdargan** [comment]
More info: I think it's a networking issue. I'm running a docker container but can't access it outside the console. I.e. the host can't access the docker image.
